pkg/secrets/secret.go

```diff
@@ -252,15 +251,15 @@ func generateBasic(req router.Request, appInstance *v1.AppInstance, secretName s
 
 	for i, key := range []string{corev1.BasicAuthUsernameKey, corev1.BasicAuthPasswordKey} {
 		if len(secret.Data[key]) == 0 {
-			// TODO: Improve with more characters (special, upper/lowercase, etc)
-			v, err := randomtoken.Generate()
+			v, err := GenerateRandomSecret(16)
```

Comment:
nit: the function that this replaces generated a string of length 54, so let's do that here too

```suggestion
			v, err := GenerateRandomSecret(54)
```

Reply:
Even if I make the suggested changes, they won't have any impact since the subsequent code truncates the length to 8 characters for the username and 16 characters for the password.